README.md

```diff
@@ -121,6 +122,57 @@ The transformer will also generate a file named `utils.js` inside the wrapper cl
 
 Note that webidl2js works best when there is a single transformer instance that knows about as many files as possible. This allows it to resolve type references, e.g. when one operation has an argument type referring to some other interface.
 
+### `[CEReactions]` and `[HTMLConstructor]`
+
+By default webidl2js ignores HTML Standard-defined extended attributes [`[CEReactions]`](https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions) and [`[HTMLConstructor]`](https://html.spec.whatwg.org/multipage/dom.html#htmlconstructor), since they require detailed knowledge of the host environment to implement correctly. The `processCEReactions` and `processHTMLConstructor` hooks provide a way to customize the generation of the wrapper class files when these extended attributes are present.
+
+Both hooks have the signature `(code) => replacementCode`, where:
+
+- `code` is the code generated by webidl2js normally, for calling into the impl class.
+
+- `replacementCode` is the new code that will be output in place of `code` in the wrapper class.
+
+If either hook is omitted, then the code will not be replaced, i.e. the default is equivalent to `(code) => code`.
+
+Both hooks also some utility methods that are are accessible via `this`:
```

Comment:
Maybe also mention the variables that are in scope for the replacement code? e.g., `globalObject` being the object passed to `install()`. (Though I don’t find any other variables being consistent throughout the different contexts.)

Reply:
Missing a “have” after “also”? Duplicate “are”?